website/src/components/dashboard/DashNavBar.jsx

```diff
@@ -81,27 +81,25 @@ class NavBar extends React.Component {
         titleStyle={styles.title}
         iconStyleRight={styles.userElement}
         iconElementLeft={
-          <Link to="/home">
+          <Link to="/">
             <IconButton><Home color="#e74c49" /></IconButton>
           </Link>
         }
         style={styles.appbar}
       >
-        <Link to="/profile">
-          <FlatButton
-            style={styles.navBtn}
+        <FlatButton
+          style={styles.navBtn}
```

Comment:
Since nothing happens when you click on this now can you change it so the cursor is default i.e. {cursor: default}